README.md

```diff
@@ -55,7 +56,7 @@ packages.
 
  * Save and exit.
 
- * Run `make`
+ * Run `make` (for a real benchmarking setup do *not* use `-j`)
```

Comment:
Then you are a brave man! `-j` is dangerous on many makefiles which do things they shouldn't.